infrastructure-aws-development.yml

```diff
@@ -0,0 +1,13 @@
+instance_groups:
+- name: master
+  jobs:
+  - name: kubernetes-minion
+    properties:
+      aws:
+        cluster-tag: kubernetes-development
+- name: minion
+  jobs:
+  - name: kubernetes-minion
+    properties:
+      aws:
+        cluster-tag: kubernetes-development
```

Comment:
Did fixing the cluster tag fix the volume mounting problem?  If not what was it -- I want to understand that failure mode so we are sure it won't happen in prod